src/main/webapp/script.js

```diff
@@ -98,21 +99,20 @@ function loadHeatmap(data) {
     aqLayer.setMap(map);
 }
 
-function searchLocation() {
-    const locationString = document.getElementById("location-search-bar").value;
-
+function selectLocation(place) {
     const locationInfo = document.getElementById("location-info");
-    locationInfo.style.visibility = "visible";
+    locationInfo.style.visibility = "visible"; 
+    
+    searchMarker.setPosition(place.geometry.location);
+    searchMarker.setVisible(true);
 
-    if (locationString !== "wagga wagga") {
-        locationInfo.innerHTML = "The only location supported by this prototype is `wagga wagga`";
-        return;
-    }
+    locationInfo.innerHTML = buildLocationHTML(place);
+}
 
+function buildLocationHTML(place) {
     htmlString = ""
```

Comment:
Local variable defined without `let`/`const`.

However, it seems this function doesn't add much value; if it would gone, I suspect the difference would only be:

```
   locationInfo.innerHTML = buildLocationHTML(place);
```
becomes:
```
   locationInfo.innerHTML = `<h3>${place.name}</h3>`;
```

Reply:
It may become more useful to have a separate function later, but because there is so little in it and that there is no certainty in whether more will be added, I have removed that function.